src/core/RenderTarget.js

```diff
@@ -37,20 +37,30 @@ class RenderTarget extends EventDispatcher {
 
 		}
 
+		const {
+			generateMipmaps = false,
+			internalFormat = null,
+			minFilter = LinearFilter,
+			depthBuffer = true,
+			stencilBuffer = false,
+			depthTexture = null,
+			samples = 0
+		} = options;
```

Comment:
This is actually destructuring. Not a super fan of this syntax. How about using `Object.assign()` instead?

Reply:
> How about using `Object.assign()` instead?

Thank you for your reply.

I'm sorry, I don't know what you mean, I wonder if you can give the sample code.

Reply:
Check out how `DRACOExporter` defines its default options:

https://github.com/mrdoob/three.js/blob/2c8cfdfc334f5758ee1facd2d95b75e33cd77b18/examples/jsm/exporters/DRACOExporter.js#L22-L32

Reply:
> Not a super fan of this syntax.

Why not? It seems clearer and simpler to me...